### PR TITLE
added scoreboard caching

### PR DESCRIPTION
**Pull request by BenDoan**

### CODE REVIEW

File: code_court/courthouse/uwsgi.ini [hunk 1]
```diff
@@ -1,8 +1,27 @@
 [uwsgi]
 module = wsgi
-http-socket = :9191
+http = :9191
+
+; http = :80
+; thunder-lock = true
+; uid = ben
+; gid = ben
```
Comment (spacetag) [inline code_court/courthouse/uwsgi.ini]: Remove commented configurations?

**Comment by BenDoan** [comment]
These are the production http settings. I should see if I can enable them
by env var or something, but for now I'm just leaving them in so I can
uncomment when running in prod.

File: code_court/courthouse/uwsgi.ini (hunk 1 above)
Comment (BenDoan) [reply code_court/courthouse/uwsgi.ini]: These are the production http settings. I should see if I can enable them
by env var or something, but for now I'm just leaving them in so I can
uncomment when running in prod.

**Comment by nrichman** [comment]
go to bed

**Comment by BenDoan** [comment]
How can I go to bed when there's code left to be written?!

On Sat, Sep 23, 2017 at 11:24 PM, Nicholas Richman <notifications@github.com

**Comment by nrichman** [comment]
Beiritifhnejwjwodiejwnw

**Pull request merged**